File: src/cli/tests/fixtures/doc/doc_text_lexicon_budgets.erl
%% Loads the configured data set from disk, parses every record
%% against the schema, resolves relative paths, retries transient
%% failures, and logs a one-line summary once the load settles
%% into a steady state operators rely on for every run of the tool.

quiet() ->
    <<"not a comment: this binary literal tail runs far past the eighty char budget">>.

%% this trailing note line deliberately runs past the eighty character budget limit for text002 warnings to fire on it

File: src/cli/tests/fixtures/doc/doc_text_lexicon_budgets.js
// Loads the configured data set from disk, parses every record
// against the schema, resolves relative paths against the config
// directory, retries transient failures with bounded backoff, and
// logs a one-line summary once the load settles into a steady state.

function quiet() {
  return 1;
}

/**
 * Parses the given payload text into a record by splitting every field,
 * validating each fragment against the configured schema, resolving
 * forward references, and folding the normalized fragments back into
 * one record value that callers can query by key or by field name.
 */
function parse(payload) {
  return payload;
}

// this trailing note line deliberately runs past the eighty character budget limit for text002 warnings to fire on it
